src/views/migrationpolicies/list/components/MigrationPolicyCreateForm/MigrationPolicyCreateForm.scss

```diff
@@ -0,0 +1,26 @@
+.migration-policy__form {
+  max-width: 50%;
+}
+
+.migration-policy__form-body {
+    margin: 1.5rem 0 0;
+    padding: 0 1rem 1.5rem;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+}
+
+.migration-policy__form-header {
+    margin-top: 1.5rem;
+    padding: 0 1rem;
+    align-items: baseline;
+    display: flex;
+    justify-content: space-between;
+    @media (min-width: 1200px) {
+        padding-left: 1.5rem;
+        padding-right: 1.5rem;
+    }
+}
+
+.migration-policy__form-header-link {
+    font-size: 14px;
```

Comment:
Font-size variables: https://www.patternfly.org/v4/utilities/text/#font-size

Reply:
I generally agree with you that we should use patternfly vars for sizes, this class is pretty much copy paste from the console repo so I can keep to be persistent with the console app design of forms.
I can check and see if any of PF vars can fit here and for the comment you made below. Thanks :)